Flepper.QueryBuilder/FlepperQueryBuilder.cs

```diff
@@ -115,5 +115,146 @@ public static ISelectCommand SelectWithDistinct(params SqlColumn[] columns)
         /// <returns></returns>
         public static ISoftDeleteCommand SoftDelete<T>() where T : class
                     => new QueryBuilder().SoftDeleteCommand<T>();
+
+        /// <summary>
+        /// Create QueryBuilderParamenter
+        /// </summary>
+        /// <typeparam name="T">class type that contain the flag</typeparam>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        public static IQueryBuilderParameter NullValue<T>(T value)
```

Comment:
Refactor this method, and remove the others below
```csharp
public static IQueryBuilderParameter NullValue<T>(Nullable<T> value) where T: struct
    => new QueryBuilderParameter<T>(value);
```

Reply:
I did not understand? Please explain why?

Reply:
This also does not work, error: `The type 'T' must be a non-nullable type in order to use it as the 'T' parameter in the generic type or method 'Nullable <T>'`

Reply:
@AlbertoMonteiro 

Could you explain why? 

Note: it will not work this way the Text type (`string`)

Reply:
Sure, I can explain, look code below:

```csharp
/// <summary>
///     Create QueryBuilderParamenter
/// </summary>
/// <typeparam name="T">class type that contain the flag</typeparam>
/// <param name="value"></param>
/// <returns></returns>
public static IQueryBuilderParameter NullValue<T>(T? value = default) where T : struct
    => new QueryBuilderParameter<T?>(value);

/// <summary>
///     Create QueryBuilderParamenter String
/// </summary>
/// <param name="value"></param>
/// <returns></returns>
public static IQueryBuilderParameter NullValueString(string value = null)
    => new QueryBuilderParameter<string>(value);
```

If you use this code, it will work for Nullable types, and string, that is a reference type!

About the `NullValue` method, you make it generic, and with the `where T : struct` we created a constraint that only accept types that are structs like int, DateTime, byte and so on...
Using this way, doest not need to create a bunch of overloads of all base struct types.

I would like you to explain, why you thumbs down my comment!!!

Reply:
Okay